infra/helper.py

```diff
@@ -1149,10 +1151,16 @@ def download_corpora(args):
       return False
 
   if args.fuzz_target:
-    fuzz_targets = [args.fuzz_target]
+    fuzz_targets = args.fuzz_target
   else:
     fuzz_targets = _get_fuzz_targets(args.project)
 
+  if not fuzz_targets:
+    logging.error(
+        'Fuzz targets not found. Please find them in %s/build.sh and specify '
```

Comment:
Might be better to recommend just doing a `python helper.py build_fuzzers`, so that line 1156 can automatically find them based on the build.

Relying on `build.sh` is error prone, and in some cases it can be very difficult to find this just by reading it (e.g. if the actual build scripts are upstream).